samples/attested_tls/server/enc/openssl/public.pem

```diff
@@ -0,0 +1,39 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
This is identical to the private key.

Reply:
the earlier mbedtls implementation also have same pattern [here](https://github.com/openenclave/openenclave/tree/master/samples/attested_tls/server/enc). Could that be because of mistake? Do i need to replace with newly generated pubic , private keys. Also the alternative we can follow is generate them during build process, instead of checking in static files. @yentsanglee @qiucwang

Reply:
The public keys are used to compare against signer_id after verification but the comparisons use the public keys defined in `OTHER_ENCLAVE_PUBLIC_KEY`. Both public.pem files are not used except the contents were copied-n-pasted to tls_client_enc_pubkey.h and tls_server_enc_pubkey.h.

Even thought this public.pem is not used but it's clearly incorrect. You may generate new key pairs but remember to replace the content in `OTHER_ENCLAVE_PUBLIC_KEY`.